woocommerce/woocommerce
# Improvements to core profiler checkbox UX

### Changes proposed in this Pull Request:

Closes #50112

Fixes the vertical alignment of checkbox and UX improvements to the plugins step

Before:

https://github.com/user-attachments/assets/8e9c5bba-6d12-44e0-b2c5-43fcb28173cb

After:


https://github.com/user-attachments/assets/6f8c7180-58ae-4704-b2eb-669ac10fa957





### How to test the changes in this Pull Request:

1. In a fresh site, go to Core Profiler
2. Observe the `I agree to share...` checkbox is vertically centered
3. Continue until the `Tell us a bit about your store` step
2. Observe the `Opt-in to receive tips...` checkbox is vertically centered
3. Continue until the `Get a boost with our free features` step 
4. Hover over a plugin card with `Installed` label
5. Observe the cursor is "Arrow" since it's not clickable
4. Hover over a plugin card without `Installed` label
5. Observe the cursor is "Pointer"
6. Click on anywhere on the card except its checkbox
7. Observe the checkbox is toggled checked/unchecked
8. Click on `Learn More` in any of the available plugins card
9. Observe a page is opened in a new tab and the checkbox is not toggled




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
